episodes/03-collaborative-git.md

```diff
@@ -0,0 +1,102 @@
+
+---
+title: Collaborative Version Control
+teaching: 180
+exercises: 2
```

Comment:
This should be minutes that it takes to solve exercises, not the number of exercises ;)

Reply:
true, 2 min is a bit short ;)